packages/webpack/mixins/start/mixin.core.js

```diff
@@ -27,10 +27,15 @@ class WebpackStartMixin extends Mixin {
               'Experimental: Enable faster development mode (modern browsers only)',
             type: 'boolean',
           },
+          fastRefresh: {
```

Comment:
We should add the option to `develop` too.

Reply:
`start` invokes `develop`, when called in dev env, and passes on the CLI-options.

Reply:
Okay, I stand corrected. Nothing's passed on to `develop`! I'd fix that in a different PR, because it also affects fast-dev & experimental-esbuild.